easybuild/easyconfigs/b/BLAST+/BLAST+-2.17.0-gompi-2025a.eb

```diff
@@ -0,0 +1,64 @@
+# #
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of
+# the policy: https://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# #
+# Author: Jan André Reuter (JSC)
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.17.0'
+
+homepage = 'https://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'gompi', 'version': '2025a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://ftp.ncbi.nlm.nih.gov/blast/executables/%(namelower)s/%(version)s/']
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+patches = ['BLAST+-2.17.0_fix-removed-boost-dummy-cond.patch']
+checksums = [
+    {'ncbi-blast-2.17.0+-src.tar.gz': '502057a88e9990e34e62758be21ea474cc0ad68d6a63a2e37b2372af1e5ea147'},
+    {'BLAST+-2.17.0_fix-removed-boost-dummy-cond.patch':
+     'c0bed1448441f3c7582357053cb5d38fcdcdfaf45d0c4a4c04f315d22ce5574d'},
+]
+
+builddependencies = [('cpio', '2.15')]
+
+dependencies = [
+    ('zlib', '1.3.1'),
+    ('bzip2', '1.0.8'),
+    ('zstd', '1.5.6'),
+    ('PCRE', '8.45'),
+    ('Boost', '1.88.0'),
+    ('GMP', '6.3.0'),
+    ('libpng', '1.6.48'),
+    ('libjpeg-turbo', '3.1.0'),
+    ('LMDB', '0.9.31'),
+    ('SQLite', '3.47.2'),
+]
```

Comment:
checking the `ldd` output of the binaries/libraries, I think we're missing runtime dependencies on:
* `libelf`
* `PCRE2`
* `util-linux`

Reply:
Are you sure? I done `ldd bin/* | less` and I could not find any of the three libs you mentioned. How did you find this? I checked both `bin` and `lib` on my laptop where `libelf` is installed, using `ldd bin/* | grep libelf` and `grep libelf lib/*` and that did not show anything.

Reply:
From the `configure --help | grep -i pcre`:
```
 --with-pcre=DIR         use PCRE installation in DIR
 --without-pcre          use internal copy of PCRE
 --with-pcre2=DIR        use PCRE2 installation in DIR
 --without-pcre2         do not use PCRE2
```
The list from that output is really long to be honest.